packages/core/src/components/Wrapper/Wrapper.tsx

```diff
@@ -1,10 +1,13 @@
 import styled from '@emotion/styled';
+import { CommonProps } from '../..';
```

Comment:
I suggest using another way of import (throughout the project): 
`import { CommonProps } from '@types/emotion';`

I'm unsure whether the `'../..'` will work in all cases.